.gitlab-ci.yml

```diff
@@ -0,0 +1,30 @@
+# See https://docs.gitlab.com/ce/ci/yaml/README.html for the documentation
+
+package:
+  image: debian:10
+  stage: build
+  before_script:
+    - apt update
+    - >
+        apt install --yes
```

Comment:
Consider using apt-get (see corresponding comment in tango-doc PR).

Reply:
Done.